src/cryptonote_core/tx_pool.cpp

```diff
@@ -1060,6 +1060,48 @@ namespace cryptonote
     return true;
   }
   //------------------------------------------------------------------
+  void tx_memory_pool::update_txpool_from_chain(size_t start_chain_height, size_t end_chain_height){
+    MGINFO_YELLOW("Starting from block " << start_chain_height << " till block " << end_chain_height << " and removing already mined txis from txpool");
+    std::vector<crypto::hash> txvector;
```

Comment:
As far as I can see `MGINFO_YELLOW` is hardly used, or hardly used anymore, in the Monero codebase. Usually you choose from the "semantic" macros `MINFO`, `MDEBUG`, `MERROR` and so on, so you people can use the `set_log` command and choose the detail / severity level they want to see.

Reply:
Thanks for the comment. This is just for debugging and temporary.

Reply:
Well, if you make it `MDEBUG` you don't even have to remove it. Somebody else may want to debug something here in a year's time, and the message may be useful again, no?

The codebase is full with such messages, and I do think they are useful and can stay.

Reply:
Yes, you are right. Final PR will of course have detailed debug messages ( and no `MGINFO_YELLOW` ).